qacode/core/enums/enum_base.py

```diff
@@ -13,9 +13,9 @@ class EnumBase(Enum):
     @classmethod
     def get_properties(cls):
         """Return enum values"""
-        return [item.value for item in type(cls)]
+        return [name for name, member in cls.__members__.items()]

```

Comment:
Esto se podría simplificar usando `keys`.
```suggestion
        return list(cls.__members__keys())
```